provisioning/miniprov/hare_mp/main.py

```diff
@@ -53,8 +57,27 @@ def execute(cmd: List[str]) -> str:
     return out
 
 
-def setup_logging():
-    logging.basicConfig(level=logging.INFO, format='%(message)s')
+def create_logger_directory():
+    """Create log directory if not exsists."""
+    _logger_directory = os.path.join(log_directory)
+    if not os.path.isdir(_logger_directory):
+        try:
+            os.mkdir(_logger_directory)
+        except BaseException:
+            raise Exception(f"{_logger_directory} Could not be created")
+
+
+def setup_logging() -> None:
+    console = logging.StreamHandler(stream=sys.stderr)
+    fhandler = logging.handlers.RotatingFileHandler(log_file,
+                                                    mode='a',
+                                                    maxBytes=5242880,
```

Comment:
- Please add an option argument to the hare_setup utility `--log-file-size`, define a constant for a default log_file_size.
- Let setup_logging() take `log-file-size` as an argument, set it default to `DEFAULT_LOG_FILE_SIZE`.

Reply:
Actually, let's not pass it as an argument but let's have a /etc/sysconfig/hare file for all the hare settings parameters. For now please ignore we can do it in a separate task.

Reply:
Okay Sure. We will do in separate task.